supervisor/resolution/const.py

```diff
@@ -37,6 +38,7 @@ class IssueType(str, Enum):
 
     FREE_SPACE = "free_space"
     CORRUPT_DOCKER = "corrupt_docker"
+    CORRUPT_REPOSITORY = "corrupt_repository"
```

Comment:
```suggestion
    CORRUPT_GIT_REPOSITORY = "corrupt_git_repository"
```

Reply:
The idea behind it is to hold it open and short. Like CORRUPT_DOCKER, possible to split into CORRUPT_OVERLAY_DOCKER, CORRUPT_NETWORK_DOCKER, etc but in the end, it doesn't maters, just CORRUPT_DOCKER is enough to fix it. So to hold it short, we could use `CORRUPT_GIT`, but not all store repo is git repository but all are Repository objects, which route the fix to correct underlay object.

Reply:
IMHO, the issue type should be rather specific so that we can identify what actually is wrong with the system. The resolution to it can be more generic, as a specific solution (e.g. recreate git repository) can fix multiple issues (corrupt git repository, missing git repository...)

Reply:
When we need to differentiate the different types of issues, I would agree, but that is not the case here, I'm fine with it being generic where it's currently used, if we need more separation in the future we can always add/change it up :+1: